src/components/Loading/index.jsx

```diff
@@ -1,6 +1,43 @@
 import React from 'react'
-import LoadingStyled from './styled'
+import { makeStyles } from '@material-ui/core/styles'
+import CircularProgress from '@material-ui/core/CircularProgress'
 
-const Loading = () => <LoadingStyled>Loading...</LoadingStyled>
+const useStyles = makeStyles(theme => ({
+  progress: {
+    margin: theme.spacing(2)
+  }
+}))
+
+// positions loader in the center of the containing div element
+// enlarges animation for better visibility
+const loaderStyle = {
+  position: 'absolute',
+  top: '0',
+  bottom: '0',
+  left: '0',
+  right: '0',
+  margin: 'auto',
+  transform: 'scale(2)'
+}
+
+const name = 'Loading Component'
+
+// sizes wrapper to viewport dimensions
+const wrapperStyle = {
+  width: '100vw',
+  height: '100vh'
+}
+
+const Loading = () => {
```

Comment:
This component does not follow our standards where it should use styled components and material-ui.

```jsx
import React from 'react'
import { CircularProgress } from '@material-ui/core';
import LoadingStyled from './styled'

function Loading () {
  return (
    <LoadingStyled>
      <CircularProgress />
    </LoadingStyled>
  )
}

export default Loading

```